resources/migrations/20220317055918-create-question.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS question (
```

Comment:
Having an IF NOT EXISTS in these CREATE commands means that the outcome of successfully running these migrations will not guarantee that the `question` table exists with these columns, just that it exists. Additionally, if the `question` table already exists, that probably isn't something we want to silently accept and move on. We should prefer that the migration fails in that situation so that we know that a table with the same name somehow already exists.

Reply:
I'll fix it when I start the next story. All tables have this issue. Btw, how do we fix this without editing the migration file? In this case I can get away with it, but I wonder how to simulate this in a more "real" project.

One strategy I can think of: repeat all the migrations I have, but without the conditionals.

Reply:
Don't change the migrations merged to master. IIRC, the migration system will recognize that a previous migration has been edited and throw up. So it isn't something one can do anyway.